doc/reference/reference_lua/config.rst

```diff
@@ -121,89 +134,204 @@ config API
 
     .. _config_api_reference_info:
 
-    ..  method:: info()
+    ..  method:: info([version])
 
         Get the current instance's state in regard to configuration.
 
-        :return: a table containing an instance's state
+        :param string version: (since :doc:`3.1.0 </release/3.1.0>`) the version of the information that should be returned. The ``version`` argument can be one of the following values:
 
-        The returned state includes the following sections:
+                               * ``v1`` (default): the ``meta`` field returned by ``info()`` includes information about the last loaded configuration
+                               * ``v2``: the ``meta`` field returned by ``info()`` includes two fields:
 
-        -   ``status`` -- one of the following statuses:
+                                   * the ``last`` field includes information about the last loaded configuration
+                                   * the ``active`` field includes information for the last successfully applied configuration
 
-            *   ``ready`` -- the configuration is applied successfully
-            *   ``check_warnings`` -- the configuration is applied with warnings
-            *   ``check_errors`` -- the configuration cannot be applied due to configuration errors
 
-        -   ``meta`` -- additional configuration information
+        :return: a table containing an instance's state. The returned state includes the following sections:
 
-        -   ``alerts`` -- warnings or errors raised on an attempt to apply the configuration
+                 -   ``status`` -- one of the following statuses:
 
-        **Examples:**
+                     *   ``ready`` -- the configuration is applied successfully
+                     *   ``check_warnings`` -- the configuration is applied with warnings
+                     *   ``check_errors`` -- the configuration cannot be applied due to configuration errors
+
+                 -   ``meta`` -- additional configuration information
+
+                 -   ``alerts`` -- warnings or errors raised on an attempt to apply the configuration
+
+        Below are a few examples demonstrating how the ``info()`` output might look.
+
+        **Example: no configuration warnings or errors**
+
+        In the example below, an instance's state is ``ready`` and no warnings are shown:
+
+        ..  code-block:: tarantoolsession
+
+            app:instance001> require('config'):info('v2')
+            ---
+            - status: ready
+              meta:
+                last: &0 []
+                active: *0
+              alerts: []
+            ...
+
+        **Example: configuration warnings**
+
+        In the example below, the instance's state is ``check_warnings``.
+        The ``alerts`` section informs that privileges to the ``bands`` space for ``sampleuser`` cannot be granted because the ``bands`` space has not created yet:
```

Comment:
```suggestion
        The ``alerts`` section informs that privileges to the ``bands`` space for ``sampleuser`` cannot be granted because the ``bands`` space has not been created yet:
```